### PR TITLE
Load images

### PR DESCRIPTION
**Pull request by erikh**
Carry of #113

### CODE REVIEW

File: roles/docker/tasks/main.yml [hunk 1]
```diff
@@ -36,6 +36,17 @@
...
     - prebake-for-dev
 
+- stat: path=/var/docker_images
+  register: docker_images
+  tags:
+    - prebake-for-dev
+
+- name:
+  shell: set -e; cd /var/docker_images; images=$(ls); for i in $images; do docker load -i $i; rm $i; done
+  when: docker_images.stat.isdir is defined and docker_images.stat.isdir
```
Comment (mapuri) [inline roles/docker/tasks/main.yml:46]: `docker_images.stat.isdir is defined` check is not required in this change.

**Comment by mapuri** [comment]
@erikh LGTM, one minor comment.

Also do you want to wait on @unclejack to confirm https://github.com/contiv/ansible/pull/113#discussion_r55555236 , as that will affect if this new task should be tagged at all.

**Comment by erikh** [comment]
Unfortunately, I really should push this through so I can finish this 
task today. I think it will be fine; if we need to patch it out we can 
(later today).

**Comment by mapuri** [comment]
sgtm, can you incorporate the last comment on `is define` check. I think we can then merge this once sanity passes.

**Comment by erikh** [comment]
already done I think; sanity is running now.

**Comment by mapuri** [comment]
thanks @erikh LGTM

**Comment by erikh** [comment]
removing the is_defined check failed, trying again with it back in there.

**Comment by erikh** [comment]
@mapuri I'm going to merge this, please tell me what to correct and I will in a future PR, I thikn this is good enough for now

**Pull request merged**